signer/src/bitcoin/utxo.rs

```diff
@@ -1082,115 +1077,63 @@ impl<'a> UnsignedTransaction<'a> {
         }
     }
 
-    /// An OP_RETURN output with the bitmap for how the signers voted for
-    /// the package.
-    ///
-    /// None is only returned if there are no requests in the input slice.
+    /// An OP_RETURN output with encoded withdrawal request IDs.
     ///
-    /// The data layout for the OP_RETURN depends on whether there are
-    /// withdrawal requests in the input slice. If there are withdrawal
-    /// requests then the output data is:
+    /// The `OP_RETURN` output has a generally-accepted 80 bytes available for
+    /// data (it is not constrained by the Bitcoin protocol itself but rather by
+    /// the miners' policy). We use this space to encode the withdrawal request
+    /// IDs.
     ///
-    /// ```text
-    ///  0       2    3     5        21            41
-    ///  |-------|----|-----|--------|-------------|
-    ///    magic   op   N_d   bitmap   merkle root
-    /// ```
+    /// Contradictory to version 0, version 1 of the OP_RETURN output will
+    /// always return an output, even if there are no requests.
     ///
-    /// A more full description of the merkle root can be found in the
-    /// comments for [`UnsignedTransaction::withdrawal_merkle_root`]. If
-    /// there are no withdrawal requests in the input slice then the layout
-    /// looks like:
+    /// ## Wire Format
+    /// The layout of the OP_RETURN output is as follows:
     ///
     /// ```text
-    ///  0       2    3     5        21
-    ///  |-------|----|-----|--------|
-    ///    magic   op   N_d   bitmap
+    ///  0       2    3                                           X<80
+    ///  |-------|----|-------------------------------------------|
+    ///    magic   op   encoded withdrawal IDs (optional+variable)
     /// ```
     ///
-    /// In the above layout, magic is the UTF-8 encoded string "ST", op is
-    /// the version byte, and N_d is the of deposits in this transaction
-    /// encoded as a big-endian two byte unsigned integer.
+    /// In the above layout:
+    /// - magic: UTF-8 encoded string "ST" (2 bytes)
```

Comment:
nit: remove the `ST` reference -- we currently use `T3` in prod and we were thinking about changing it

Reply:
[b02d783](https://github.com/stacks-network/sbtc/pull/1512/commits/b02d7837663c4cef165471cfb46cb1c4615b92ef)